app/workers/billing_worker.rb

```diff
@@ -62,7 +62,7 @@ def perform(buyer_id, provider_id, time)
   private
 
   def store_summary(buyer_id, billing_result)
-    summary = BillingSummary.new(bid)
+    summary = BillingSummary.new(batch && bid)
```

Comment:
@guicassolato is it normal that the batch is `nil` sometimes?
Asking because it is related to Billing and the summary.

Reply:
It comes from https://github.com/breamware/sidekiq-batch/blob/e3b293519ec697b264339dbaf1dac25fb1f69195/lib/sidekiq/batch/extension/worker.rb#L3, set in https://github.com/breamware/sidekiq-batch/blob/e3b293519ec697b264339dbaf1dac25fb1f69195/lib/sidekiq/batch.rb#L72-L78

Reply:
Sorry I explained myself incorrectly.
In which case/scenario batch can be `nil`

Reply:
`batch` (and `bid`) will be `nil` if not set in `Thread.current[:bid]` – i.e. the job not part of a batch or it's checking in different thread, I guess.